zhmccli/_cmd_port.py

```diff
@@ -29,11 +29,18 @@
 
 def find_port(cmd_ctx, client, cpc_name, adapter_name, port_name):
     """
-    Find a port by name and return its resource object.
+    Find a port by name or index and return its resource object.
```

Comment:
I guess we just leave the option to use the name for backwards compatibility? The name of a port is always "Port #index", so I see no advantage from a user perspective of using the name.

Reply:
I can provide the following reasons for supporting the port name:
* Backwards compatibility. Although I doubt that anyone has used this command because we would have gotten an issue opened due to the bug
* Consistency: We generally use object names in the zhmc parameters.
* Flexibility: The port name can be changed and can have a semantic for a user that they want to use. For example, it could be renamed to become unique at the CPC level by including the adapter name in the port name.

Reply:
On your suggested change: Very strange - my local repo clone already has the proposed text, and it is at commit eaa00d42 (same as the latest commit in the PR at the time Simon reviewed it). In any case, DONE.